### PR TITLE
refresh account settings page (bug 1103228)

### PR DESCRIPTION
**Pull request by ngokevin**
- moves settings/main.html to settings.html to match route, with all other PRs the settings template folder will be removed
- gets rid of a ton of old settings page CSS, cruft markup, and unneeded classes

# Desktop Logged In

screen shot 2014-12-29 at 1 43 49 pm

# Tablet Logged In 

screen shot 2014-12-29 at 1 43 59 pm

# Mobile Logged In

screen shot 2014-12-29 at 1 44 19 pm

# Desktop Logged Out

screen shot 2014-12-29 at 1 44 28 pm

# Tablet Logged Out

screen shot 2014-12-29 at 1 44 36 pm

# Mobile Logged Out

screen shot 2014-12-29 at 1 44 49 pm

### CODE REVIEW

**Comment by ngokevin** [comment]
@spasovski

File: src/media/css/account.styl [hunk 1]
```diff
@@ -1,230 +1,129 @@
...
-    .extras {
-        padding-bottom: 0;
-    }
 }
 
-.toggles {
-    display: table;
-    table-layout: fixed;
-    width: 100%;
+.account-settings-save {
```
Comment (spasovski) [inline src/media/css/account.styl]: Verbose name but it should help with on-boarding contributors.

File: src/media/js/views/settings.js [hunk 2]
```diff
@@ -32,35 +33,29 @@ define('views/settings',
...
             if (data.enable_recommendations) {
-                z.body.addClass('show-recommendations');
-            } else {
-                z.body.removeClass('show-recommendations');
+                // Toggle recommended nav item depending on what was checked.
+                z.body.toggleClass('show-recommendations', data.enable_recommendations);
             }
             update_settings();
-            notify({message: gettext('Settings saved')});
+            notify({message: gettext('Your settings have been successfully saved')});
```
Comment (spasovski) [inline src/media/js/views/settings.js]: Agreed.

**Comment by spasovski** [comment]
Is this alignment right?

**Comment by spasovski** [comment]
Not sure if it's this commit but header and paragraphs are misaligned here:

**Comment by spasovski** [comment]
r+wc

**Comment by ngokevin** [comment]
the alignment seems right. the dotted line doesn't start at the top of the input. if it did, it seems like it would match.

**Comment by ngokevin** [comment]
thanks

**Pull request merged**